code/index.html

```diff
@@ -1,30 +1,162 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Koffie business site</title>
+  <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="./style.css" />
+</head>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<body>
+  <!-- video or image as a header is cool :) -->
+  <header-container>
+    <header id="hero">
+      <video preload="auto" autoplay muted loop playsinline>
+        <source src="./hero-seeds-video.mp4">
+        <p>Your browser does not support the video tag.</p>
```

Comment:
Is this a message that it will be shown in case that the video can't be played right?Nice!

Reply:
Exactly, thank you :)